internals/daemon/api_notices_test.go

```diff
@@ -18,40 +18,80 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 	"time"
 
 	. "gopkg.in/check.v1"
 
+	"github.com/canonical/pebble/internals/osutil/sys"
 	"github.com/canonical/pebble/internals/overlord/state"
 )
 
+func mockSysGetuid(fakeUid uint32) (restore func()) {
```

Comment:
We usually call similar functions `fakeXyz`. For example, from `internals/osutil/export_test.go`:

```go
func FakeSyscallGetpgid(f func(int) (int, error)) (restore func()) {
	oldSyscallGetpgid := syscallGetpgid
	syscallGetpgid = f
	return func() {
		syscallGetpgid = oldSyscallGetpgid
	}
}
```